HealthcareSystem/PatientWebAppUnitTests/CreateObjectsForTests/ICreateTestObject.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PatientWebAppTests.CreateObjectsForTests
+{
+    public interface ICreateTestObject
```

Comment:
I think keeping this interface might be nice, but its real purpose would really be just to make the naming of the methods in the CreateX classes consistent. To do that I would make it generic and return the correct type.